aries-site/src/themes/aries.js

```diff
@@ -192,6 +192,11 @@ export const aries = {
     gap: 'medium',
     color: 'text-strong',
   },
+  formField: {
+    border: {
+      side: 'all',
+    },
```

Comment:
Do we need to add a `border-radius: 4px` to round out the corners of the inputs? Looks like there is a 4px round in the Figma file: https://www.figma.com/file/iXWcXa3QtXAGHnXZnONnty/hpe-design-system?node-id=0%3A1

Reply:
I couldn't find Form references in this link. Any other resource you can share?

Reply:
Try this link: https://www.figma.com/file/iXWcXa3QtXAGHnXZnONnty/hpe-design-system?node-id=149%3A107 (same file, but links to the "Input Page" which contains the form examples at the bottom)

Reply:
Actually, this link takes you directly to then example: https://www.figma.com/file/iXWcXa3QtXAGHnXZnONnty/hpe-design-system?node-id=343%3A230

Reply:
This PR is the preliminary work to take care of rounding on the theme https://github.com/grommet/grommet/pull/3730

Reply:
Since https://github.com/grommet/grommet/pull/3730 has been completed and theme has been published, can this PR be closed?

Reply:
yep